src/server/upgrade/index.md

```diff
@@ -10,6 +10,64 @@ Make sure to always back up your database data before doing a migration.
 
 [[toc]]
 
+## Migration guide from 2024.2.x to 2025.2.x
```

Comment:
This should be subsection of already existing migration guide (next to EE 2025.2.0)